### PR TITLE
DEV-1

### PR DESCRIPTION
**Pull request by ElizavetaKovalchuk**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            {
+                Console.WriteLine("Please enter a numeric argument.");
+            }
+            else
+            {
+                SequenceOfSymbol acceptedSequence = new SequenceOfSymbol(args[0]);
+                Console.WriteLine(acceptedSequence.SerchMaxNumberOfSymbol());
+            }
+           
+            Console.ReadKey();
```
Comment (AlexanderSoroka) [inline Program.cs]: Unnecessary readkey operation that will block automation

File: SequenceOfSymbol.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        public SequenceOfSymbol(string acceptedSequence)
+        {
+            sequenceOfSymbol = acceptedSequence;
+        }
+        /// <summary>
+        /// This method finds the maximum number of identical consecutive symbols in a string
+        /// </summary>
+        /// <returns>The maximum number of identical consecutive symbols in a string</returns>
+        public int SerchMaxNumberOfSymbol(){
+            char[] sequence = sequenceOfSymbol.ToCharArray();
```
Comment (AlexanderSoroka) [inline SequenceOfSymbol.cs]: there is no need to convert string to char array

File: SequenceOfSymbol.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        public string sequenceOfSymbol;
+        public SequenceOfSymbol(string acceptedSequence)
+        {
+            sequenceOfSymbol = acceptedSequence;
+        }
+        /// <summary>
+        /// This method finds the maximum number of identical consecutive symbols in a string
+        /// </summary>
+        /// <returns>The maximum number of identical consecutive symbols in a string</returns>
+        public int SerchMaxNumberOfSymbol(){
```
Comment (AlexanderSoroka) [inline SequenceOfSymbol.cs]: codestyle violation

**Review by AlexanderSoroka — CHANGES_REQUESTED** [review]

File: Program.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (IgorKheidorov) [inline Program.cs]: Remove unnecessary usings

File: SequenceOfSymbol.cs [hunk 5]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (IgorKheidorov) [inline SequenceOfSymbol.cs]: Remove

File: SequenceOfSymbol.cs [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        }
+        /// <summary>
+        /// This method finds the maximum number of identical consecutive symbols in a string
+        /// </summary>
+        /// <returns>The maximum number of identical consecutive symbols in a string</returns>
+        public int SerchMaxNumberOfSymbol()
+        {
+            int max_count = 0;
+            int count = 1;
+            for (int i = 0; i < sequenceOfSymbol.Length - 1; i++)
```
Comment (IgorKheidorov) [inline SequenceOfSymbol.cs]: Bug, input one symbol, result will be 0, has to 1

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]

File: Entry.cs [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        /// <summary>
+        /// It is an entry point of the program and passes to the console the maximum number of identical consecutive symbols in the string
+        /// </summary>
+        /// <param name="args">Сommand-line arguments</param>
+        static void Main(string[] args)
+        {
+            if (args.Length == 0)
+            {
+                Console.WriteLine("Please enter a numeric argument.");
```
Comment (AlexanderSoroka) [inline Entry.cs:15]: Wrong end-user message

File: SequenceOfSymbol.cs [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    /// This class contains methods for working with the sequence
+    /// </summary>
+    class SequenceOfSymbol 
+    {
+        public string sequenceOfSymbol;
+        public SequenceOfSymbol(string acceptedSequence)
+        {
+            sequenceOfSymbol = acceptedSequence;
+        }
+        /// <summary>
```
Comment (AlexanderSoroka) [inline SequenceOfSymbol.cs:13]: missed blank line

File: SequenceOfSymbol.cs [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+                {
+                    max_count = count;
+                    count = 1;
+                }
+                else
+                {
+                    count = 1;
+                }
+            }
+            return max_count;
```
Comment (AlexanderSoroka) [inline SequenceOfSymbol.cs:37]: if sequenceOfSymbol is empty it will return invalid value

**Review by AlexanderSoroka — APPROVED** [review]
Approved due to deadline

**Pull request merged**